include/valijson/validation_visitor.hpp

```diff
@@ -347,6 +347,81 @@ class ValidationVisitor: public constraints::ConstraintVisitor
         return numValidated > 0;
     }
 
+    /**
+    * @brief    Validate current node against a FormatConstraint
+    * 
+    * @param    constraint  Constraint that the target must validate against
+    * 
+    * @return   \c true if validation succeeds; \c false otherwise
+    */
+    bool visit(const FormatConstraint &constraint) override
+    {
+        const std::string s = m_target.asString();
+        const std::string format = constraint.getFormat();
+        if (format == "date") {
+            std::regex date_regex("^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])$");
```

Comment:
Can these regexes be made static, to avoid parsing them every time this validator is called?

Reply:
This could also benefit from a comment above each regex, to show an example of a string that it matches.

Reply:
Added some examples